jsdoc_conf.json

```diff
@@ -6,6 +6,9 @@
     "plugins"      : [],
     "recurseDepth" : 10,
     "source"       : {
+        "exclude" : [
+            "docs"
```

Comment:
It is excluded.

Reply:
But why does the PR contain so many changes to files located in that directory?

Reply:
Because the jsdoc got updated. It's pushed to that directory and then shown in GitHub pages.

Reply:
Oh I see it is the output of the job... Somehow I mixed that up